CHANGELOG.md

```diff
@@ -14,6 +14,14 @@
 
 </details>
 
+## 0.77.1 (August 2, 2018)
```

Comment:
please move this to the unreleased section above

Reply:
Sure. Will update this.